wormchain/app/params/proto.go

```diff
@@ -0,0 +1,22 @@
+package params
+
+import (
```

Comment:
I believe that we can get all of the `EncodingConfig` using `cosmoscmd.MakeEncodingConfig`. Does this work? If so, we can probably remove this file as well as `encoding.go`.

Reply:
yes, good catch, cleans things up nicely.